Rakefile

```diff
@@ -1,3 +1,5 @@
+require 'dotenv/load'
```

Comment:
This is not quite the right way to do this, since it's loading dotenv directly instead of via Bundler. You might get a different version than expected.

We could make this nicer by adding an `:environment` Rake task which is a prerequisite to all the tasks which need a third-party library (including the `import:*` tasks). The [Atlas `:environment`](https://github.com/quintel/atlas/blob/master/lib/tasks/environment.rb) might serve as an example:

```ruby
task :environment do
  require 'bundler'
  Bundler.require(:development)

  Atlas.data_dir = File.expand_path(File.dirname(__FILE__))
end

namespace :import do
  task dataset: :environment do
    # ...
  end

  task node: :environment do
    # ...
  end

  task carrier: :environment do
    # ...
  end
end
````